src/pages/setpassword/SetPasswordForm.js

```diff
@@ -0,0 +1,83 @@
+import React, {Component} from 'react';
+import lodashGet from 'lodash/get';
+import {Text, TextInput, View} from 'react-native';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import styles from '../../styles/styles';
+import ButtonWithLoader from '../../components/ButtonWithLoader';
+import {setPassword} from '../../libs/actions/Session';
+import setPasswordPagePropTypes from './setPasswordPagePropTypes';
+import ONYXKEYS from '../../ONYXKEYS';
+
+class SetPasswordForm extends Component {
```

Comment:
How did we end up here again? 😬  
We already established this could be achieved in a single file, no?